docs/integrations/containers-orchestration/kubernetes.md

```diff
@@ -79,22 +79,18 @@ Reference the [Deployment Guide](https://github.com/SumoLogic/sumologic-kubernet
 The Deployment Guide has information on advanced configurations, best practices, performance, troubleshooting, and upgrading for our latest and previous versions of supported software.
 
 
-## Installing the Kubernetes App
+## Installing the Kubernetes app
 
-Now that you have set up the collection for Kubernetes App, install the Sumo Logic App for Kubernetes to use the pre-configured Kubernetes dashboards that provide visibility into your Kubernetes environment.
+Now that you have set up the collection for Kubernetes app, install the Sumo Logic App for Kubernetes to use the pre-configured Kubernetes dashboards that provide visibility into your Kubernetes environment.
 
 To install the app, do the following:
 
-1. Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
-2. From the App Catalog, search for **Kubernetes** and select the app.
-3. Click **Add to Library**.
-4. Complete the following fields:
-   * App Name. You can retain the existing name, or enter a name of your choice for the app.
-   * Data Source. For each the sources listed, enter a Custom Data Filter or Source Category, as follows:
-     * For Falco Log Source, leave  **Source Category** selected, and enter the source category according to configuration for `sumologic.logs.container.sourceCategory` in values.yaml.
-     * For Events Log Source, leave **Source Category** selected, and for the source category, enter *events* or one that matches the source categories in your environment.
-   * Advanced. Select the location in the Library (the default is the Personal folder in the Library), or click New Folder to add a new folder.
-5. Click **Add to Library**.
+1. Locate and install the app from the **App Catalog**.
+2. Search for **Kubernetes** and select the app.
+3. Optionally, on this page, you can also see a preview of the dashboards included with the app.
+4. Click **Add Integration**.
+5. On the next configuration page, titled **Install App**, follow the instructions (Add Repo, Monitor & Alert Installation, Install Helm chart). When you're done, click **Next**.
+6. (Optional) On the next configuration step, titled **Preview & Done**, click the blue **Explore Dashboards** button or **Open Dashboards in Library** to check out your Kubernetes dashboards. Here, you'll also be given the options to add another cluster, configure Kubernetes monitors, or manage your collectors.
```

Comment:
May be we can add a warning that if you click on next without finishing the helm chart commands then Open Dashboards in library link won't work.

Reply:
To clarify, are you saying that you need to finish the Helm chart commands, or else the data won't load into the dashboards at all?

Reply:
I tried out the onboarding & found out that Helm command is responsible for the creation of the dashboards. Basically, app doesn't get installed if that command is not run. Since, the intention of user is to install the app and have dashboards, this command is a must. @himanshu219 correct me if I am wrong.

Reply:
yes that's correct

Reply:
The `helm` is mandatory step. Without it the app won't be installed.

Reply:
does this sound ok? _Helm chart installation via command line is required in order to install the Kubernetes app properly and see the dashboards in the next step._

Reply:
We can remove "properly". Rest wordings looks good to me. Make sure we add this to the instructions for the "Install App" step.